_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -478,6 +478,19 @@ std::pair<bool, bool> check_fei(VideoCORE* core)
     return std::pair<bool,bool>(true, *feiEnabled);
 }
 
+static inline bool is_sw_fallback_supported(mfxU32 codec_id, VideoCORE* core)
```

Comment:
AFAIR, `static inline` makes sense in C, why we need that `static` in C++?

Reply:
@dmitry-gurulev  AFAIK this usage of ```static``` is still valid in C++ and has the same meaning as in C. If your question is why not to use of unnamed ```namespace``` to do the same (limit the lifetime and hide from linker)... well, I have no preference between these two. ```namespace``` approach seems to be more C++, I can change approach in other PR